### PR TITLE
use pds.api-client==1.1.0

### LINKED ISSUE #1: Improved user operation and installation guides with github pages
This task is to generate github pages for the user guide using favorite Python docs generator.

- [x] Deploy pds-deep-archive to pypi for easy install
- [x] Dependencies/Assumptions  - e.g. python3 must be installed
- [x] Installation Guide - let's start with buildout instructions then have alternative instructions using virtualenv since most of our users will not be able to do that
- [x] Operation Guide - very basic "this is how you run it" with an output of `-h`

### LINKED ISSUE #2: Implement SIP generation offline capabilities
Formal requirements to come (#3), but for now, we need to get something together to accomplish this.

Informal requirements:
1. Software shall produce SIP Manifest table (and eventually AIP manifest (#2)) with accompanying label
2. The SIP Manifest should be a tab-separated 4 column manifest table with columns, checksum, checksum_type, file_specification_url, LIDVID (see example ladee_mission_bundle_SIP_4col_manifest_v1.0.xml and ladee_mission_bundle_SIP_4col_manifest_v1.0.tab in attached .zip)
3. A user will input a bundle directory, and the software will read the bundle.xml at the top level of the bundle. The software then proceed to query the registry for all *primary* products associated with that bundle (using the bundle lidvid) (not sure if this is possible, if not. we may need to parse all the collection lidvids/lids from the bundle.xml and use those to query the registry).
4. From the registry output, we should parse the necessary fields to populate the SIP Manifest.
5. More to come...

Use https://pds-dev-el7.jpl.nasa.gov/services/registry/pds/select?q=*%3A* for testing.

Example Python implementation, SIP/AIP products, templates, etc: [aaaZip_NSSDC_20190820.zip](https://github.jpl.nasa.gov/PDSEN/aip-gen/files/5301/aaaZip_NSSDC_20190820.zip)
Test bundle: https://atmos.nmsu.edu/PDS/data/PDS4/LADEE/mission_bundle/ (fixed URL)

### LINKED ISSUE #128: Upgrade to latest PDS API Client
## 💡 Description

https://github.com/NASA-PDS/pds-api-client/releases/latest

## Blocked by
Tagging new pds-api-client release per https://github.com/NASA-PDS/software-issues-repo/issues/30

### PR DESCRIPTION
**Pull request by tloubrieu-jpl**
## 🗒️ Summary
As said in the title

## ⚙️ Test Data and/or Report
Tested with registry 1.0.1 deployed as describe on https://nasa-pds.github.io/registry/install/docker-compose.html

    pip install -e .
    pds-deep-registry-archive --url http://localhost:8080 --site PDS_ATM urn:nasa:pds:insight_rad::2.1

## ♻️ Related Issues

Fixes #128

### CODE REVIEW

**Review by nutjob4life — COMMENTED** [review]
Definitely going in the right direction! Curious to see how `mypy` will react without the type-ignoring directives on the import statements.

**Comment by nutjob4life** [comment]
@tloubrieu-jpl, I thought I had reviewed this but I guess there's an outstanding question of `mypy` type-checking.

File: src/pds2/aipgen/registry.py [hunk 1]
```diff
@@ -56,13 +56,12 @@
 # the name of the ``model`` package is ``model``, singular. But then seemingly at random, it becomes
 # ``models`` plural. And even some releases support *both*. So here we try to accomodate whatever the
 # flavor du jour is.
-try:
-    from pds.api_client.model.product import Product  # type: ignore
-except ImportError:
-    from pds.api_client.models.product import Product  # type: ignore
+
+from pds.api_client.model.pds_product import PdsProduct
```
[the remark this answers is not in the document]
Comment (jordanpadams) [reply src/pds2/aipgen/registry.py]: @nutjob4life done

File: src/pds2/aipgen/registry.py [hunk 2]
```diff
@@ -56,13 +56,12 @@
...
 
 # If this fails to import, then we're using a pds.api-client ≤ 0.5.0, which I'm arbitrarily declaring "too old":
 from pds.api_client.exceptions import ApiAttributeError  # type: ignore
+from pds.api_client.exceptions import NotFoundException
```
[the remark this answers is not in the document]
Comment (jordanpadams) [reply src/pds2/aipgen/registry.py]: @nutjob4life done

**Comment by jordanpadams** [comment]
2 requested changes completed. merging.

**Pull request merged**